### DISCUSSION TITLE
log analyzer

### DISCUSSION THREAD

**Discussion by llcfhllml** [FoobarOy/foomuuri#25] [Ideas]
Hi 
exist a tool to analyze nftables and foomuuri logs ?

for example a parser of syslog with the minimal information like 

time src-int dest-int  source-ip dest-ip  protocol src-port dest-port 

12:25:08  ens19  ens18     77.90.185.123   2.3.253.123  TCP  44116 63383

**Comment by kimheino** [comment]
Sorry, I don't know any log analyzer. I have never needed one. Foomuuri's logging format is quite generic so most nftables-analyzers should work.

**Comment by ajayramaswamy** [comment]
I have been using fwlogwatch for years and it still works fine.
http://fwlogwatch.inside-security.de/
